openshift/console
# Add test scenario for project status page

This PR adds test scenarios for:

- **Check all-projects from project selection list.**
![allProjectsSelection](https://user-images.githubusercontent.com/14801299/56197516-9dfecf00-606b-11e9-9754-46b0bab70dd8.png)




- **Check Software Info for Kubernetes version.**
![projectsStatusPage](https://user-images.githubusercontent.com/14801299/56197463-760f6b80-606b-11e9-89ae-050c348325a1.png)

local pass log is available [here](http://pastebin.test.redhat.com/754693)
@spadgett @jhadvig Can you please help review?
